html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Page Title</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <div>
+    <form action="index.html"/>
```

Comment:
There should be only one `form` element holding all the content for the entire form, not one `form` element surrounding each form control element

Reply:
![image](https://user-images.githubusercontent.com/117887912/206051274-c67109f2-78ea-41be-bb76-f2ccea3feaff.png)
![image](https://user-images.githubusercontent.com/117887912/206051310-ac6c1fd2-6c6c-4c40-bc98-41d0d3ce9551.png)